packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -495,7 +495,7 @@ export const _generateStyles = (
             active: {
                 color: light ? fadedColor : activeColor,
                 ":after": {
-                    height: 1,
+                    height: theme.size.height.tertiaryHover,
```

Comment:
Using the [same token as the height of the underline in the hover state](https://github.com/Khan/wonder-blocks/pull/2259/files#diff-ede6855e27a1ecfe88d6f2e68ba1979e36ea896776f7e023078b353df3b84b94R479)!